### DISCUSSION TITLE
Using @ to mention another user is not consistent across platforms

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#1032] [Product Feedback]
When the user types "@" on mobile a list of options shows on Android. 

On iOS and Web, the user must type "@" followed by a character. 

We should align on which we'd like to implement and the create the issues on the corresponding platforms to make that the case.

Reference: https://github.com/vector-im/element-web/issues/24018

In web, this looks like:

<img width="348" alt="Screenshot 2023-02-22 at 11 24 41" src="https://user-images.githubusercontent.com/89144281/220606618-0bd935b1-ca4c-4fde-a6af-d88e85315639.png">

**Comment by daniellekirkwood** [comment]
@jakewb-b did y'all address this question as part of the RTE work? Have we already reached a conclusion on this - might be an easy one to close out?

@VolkerJunginger & @amshakal Do you have thoughts? Once a conclusion has been reached we should implement it in EX only (and not the classic apps)

My personal preference is to only show the drop-down/up when the user has typed "@" and a letter as this is likely to be better for performance in large rooms.

**Comment by daniellekirkwood** [comment]
Decision made @jakewb-b & @daniellekirkwood 22/02/2023: 

- All platforms should wait for first character before showing a list of available options
- We will open a discussion that is an enhancement to build a "suggested" list to show before the user has selected a letter. This list could include @Room and @Recents for example

[Internal conversation reference](https://matrix.to/#/!UCBgpJmTgAHYpcJBoh:matrix.org/$yZL6l2CsIS_b8bjmI5uNdfeSffOPoRVmbaFmbBePYdM?via=matrix.org&via=element.io&via=one.ems.host)

**Reply by amshakal** [reply]
This seems like a sound solution to me

**Comment by daniellekirkwood** [comment]
Issue created: https://github.com/vector-im/element-android/issues/8156
